velox/docs/functions/uuid.rst

```diff
@@ -0,0 +1,7 @@
+=============================
+UUID Functions
+=============================
+
+.. function:: uuid() -> varchar
+
+   Returns a pseudo randomly generated UUID as a single character string
```

Comment:
In Presto, uuid function returns the value of type UUID not VARCHAR.

Reply:
@mbasmanova  Add uuid type now, thanks for your review!